charts/substra-backend/templates/deployment-server.yaml

```diff
@@ -99,6 +95,8 @@ spec:
             mountPath: /var/substra/servermedias
           - name: statics
             mountPath: /usr/src/app/backend/statics
+          - name: uwsgi-socket
+            mountPath: /usr/src/app/backend/socket
```

Comment:
Minor: /usr/src is an uncommon folder for a socket, isn’t it?
 What about a more standard folder, like /var/substra ?

Reply:
Yes !

Reply:
sockets usually live in `/var/run/xxx.sock` if you are looking for something standard ;)

Reply:
Yes, good point !